src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -31,6 +31,11 @@ public partial class PopupWindow : Popup
         //Assembly path to the Resources folder
         private const string resourcesPath = "Dynamo.Wpf.Views.GuidedTour.HtmlPages.Resources";
 
+        /// <summary>
+        /// This property will be hold the path of the WebView2 cache folder, the value will change based in if DynamoSandbox is executed or Dynamo from Revit is executed
```

Comment:
consider this generally, it's not just DynamoRevit, what about DynamoForCivil or Dynamo Formit etc?

Reply:
Comment updated: https://github.com/DynamoDS/Dynamo/pull/13245/commits/45e845bef4b113c5ebd934b2925f71e578253fab